src/crimson/os/seastore/extent_placement_manager.cc

```diff
@@ -0,0 +1,157 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:nil -*-
+// vim: ts=8 sw=2 smarttab expandtab
+
+#include "crimson/os/seastore/journal.h"
+#include "crimson/os/seastore/extent_placement_manager.h"
+#include "crimson/os/seastore/lba_manager/btree/lba_btree_node_impl.h"
+#include "crimson/os/seastore/omap_manager/btree/omap_btree_node_impl.h"
+#include "crimson/os/seastore/onode_manager/staged-fltree/node_extent_manager/seastore.h"
+#include "crimson/os/seastore/collection_manager/collection_flat_node.h"
+#include "crimson/os/seastore/object_data_handler.h"
+
+namespace {
+  seastar::logger& logger() {
+    return crimson::get_logger(ceph_subsys_seastore);
+  }
+}
+
+namespace crimson::os::seastore {
+
+SegmentedAllocator::SegmentedAllocator(
+  SegmentProvider& sp,
+  SegmentManager& sm,
+  Cache& cache)
+  : segment_provider(sp),
+    segment_manager(sm),
+    cache(cache)
+{
+  for (unsigned i = 0;
+       i < crimson::common::get_conf<uint64_t>(
+	 "seastore_init_rewrite_segments_num_per_device");
```

Comment:
@cyx1231st Here is the usage

Reply:
Oh sorry, missed that we have multiple rewriters.